### ISSUE TITLE
Remove clientside JS popup after accepting workshop 

### ISSUE THREAD

**Issue by ankitjavalkar** [FOSSEE/workshop_booking#97]
The client side Javascript displays an alert (in the browsers native GUI modal) stating: ```Refreshing Page please wait, if you don't see your changes, please refresh this page again.```

**Comment by CruiseDevice** [comment]
I am adding more to the issue. The functionality uses ajax, which is not required. It needs to be replaced entirely with standard Django view POST request.

**Comment by pnshiralkar** [comment]
Should this be done for change date and delete workshop also?

**Comment by CruiseDevice** [comment]
Yes. The popup is one part, but the entire functionality is ajax. We need to change that ajax functionality to normal Django view POST request.
